tigerpath/majors_and_certificates/scripts/verifier.py

```diff
@@ -316,6 +318,65 @@ def _format_courses_output(courses):
                 output[i][j]["settled"] = course["settled"]
     return output
 
+def _parse_year_code(code, year):
```

Comment:
I don't think this is a very descriptive name. It would be better if it evoked what the return value is -- maybe something like `_year_matches_code`?

Reply:
Good point! 
In that case, I'm also reversing the argument order to match the new name, so it's not confusing on that front.

```suggestion
def _year_matches_code(year, code):
```